src/test/blockmanager_tests.cpp

```diff
@@ -21,23 +21,26 @@ BOOST_FIXTURE_TEST_SUITE(blockmanager_tests, BasicTestingSetup)
 BOOST_AUTO_TEST_CASE(blockmanager_find_block_pos)
 {
     const auto params {CreateChainParams(ArgsManager{}, CBaseChainParams::MAIN)};
-    BlockManager blockman{{}};
+    node::BlockManager::Options blockman_opts{
```

Comment:
Nit: Can be `const` (here and elsewhere) and skip the `node::` namespacing, since it is imported above. If this is merged before #27125 I will follow-up there.